parent/pom.xml

```diff
@@ -517,12 +517,12 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.9</version>
+                <version>3.6</version>
```

Comment:
why the downgrade?

Reply:
In order to be still compatible with AEM 6.4. For OSGi bundles you rather reference the minimum version you are compatible with.